### ISSUE TITLE
"which" not working right in gemini3d.find.gemini_exe

### ISSUE THREAD

**Issue by julesvanirsel** [gemini3d/mat_gemini#29]
Hey, for some reason I wasn't able to use gemini3d.find.gemini_exe because "which" wasn't working right? I've changed 

```
for p = paths
  for b = bindirs
    exe = which(name, fullfile(p, b));
    if ~isempty(exe)
      exe = absolute_path(exe);
      return
    end
  end
end
```
to the following 
```
exe = "";
for p = paths
  for b = bindirs
    if exist(fullfile(p,b,name),'file') ~= 0
      exe = absolute_path(fullfile(p,b,name));
      return
    end
  end
end
```
as a possible work around. I don't know, maybe it's a windows thing...

Jules van Irsel

**Comment by mattzett** [comment]
Just a comment Jules and I spent and hour or so trying to get which() to work like it should in R2022 matlab on windows without any luck so we came up with this workaround.

**Comment by scivision** [comment]
Was this using Gemini3D in WSL from Matlab?

**Comment by scivision** [comment]
Implemented as  2d7b78a259977a in case WSL was an issue

**Comment by scivision** [comment]
OK it turns out I hadn't updated matlab-stdlib submodule for this in a while. I removed the 2d7b78a workaround to see if Mat_gemini works now for you

**Comment by mattzett** [comment]
thanks for taking a look at this @scivision

**Comment by scivision** [comment]
this is further fixed by 04f415b47
